### ISSUE TITLE
Customizer and popup options don't update on refresh

### ISSUE THREAD

**Issue by ivandotv** [ThemeFuse/Unyson#1905]
So Inside the customizer, and inside the popup, when I edit the value and hit **save** on the popup.
It triggers the customizer refresh, and the site is reloaded however, new value is not displayed.
It still shows the old value.

I need to hit the **save and publish** customizer option, *AND* after that I need to refresh the customizer (by changing some other option) and only then the new value is shown.

So basically if I have a `title` option in popup
1.I edit that option and change the title text.
2.  hit *save* inside the popup ( popup saves with success)
3. Customizer reloads the site.

.... I still get old value (title on the  site  is not changed)

4 Hit **save and publish** 
5 Reload the site inside the customizer (or in the frontend)

.... new value is shown.

**Comment by moldcraft** [comment]
Please give us the test options array so I can add it in `customizer.php` to reproduce the problem.

**Comment by ivandotv** [comment]
After a lot of debugging, I have found the issue.
It is very strange.
A custom element of *visual composer* plugin was accessing the `fw_get_db_customizer_option` function.
The function was accessed on `vcBeforeInit` plugin hook.

And the function **works** and it gets the value, but it somehow **silently** breaks the customizer.
Also there are no error logs, nothing.

Removing the call to `fw_get_db_customizer_option` removes the error.

So it appears that maybe there is some kind of *race* issue between the plugins.

Is there a way to access the customizer options (set via unyson) without using the *Unyson* framework?

If you look at the image, you will see that I get the values back.
![selection_002](https://cloud.githubusercontent.com/assets/390700/17977711/189ac0fa-6af3-11e6-9c87-622a154d9453.png)

**Comment by moldcraft** [comment]
> Is there a way to access the customizer options (set via unyson) without using the Unyson framework?

[Sure](https://github.com/ThemeFuse/Unyson/blob/v2.5.12/framework/helpers/database.php#L379) `get_theme_mod('fw_options')`
